.github/workflows/security.yml

```diff
@@ -0,0 +1,28 @@
+on: 
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**/*.md"
+  schedule:
+    - cron: 15 14 * * 6
```

Comment:
what do you say about `00 1 * * 1` schedule, we will have fresh result on mondays.